Formula/libimobiledevice.rb

```diff
@@ -20,20 +20,77 @@ class Libimobiledevice < Formula
     depends_on "libxml2"
   end
 
+  option "with-python", "Enable Cython Python bindings"
+  option "with-python3", "Enable Cython Python3 bindings"
+
   depends_on "pkg-config" => :build
   depends_on "libtasn1"
-  depends_on "libplist"
   depends_on "usbmuxd"
   depends_on "openssl"
+  depends_on "libplist" => (build.with?("python") ? "with-python" : []) || (build.with?("python3") ? "with-python3" : [])
+  depends_on :python => :optional
+  depends_on :python3 => :optional
+
+  resource "cython" do
+    url "https://github.com/cython/cython/archive/0.24.tar.gz"
+    sha256 "b60b91f1ec88921a423d5f0a5e2a7c232cdff12d9130088014bf89d542ce137b"
+  end
+
+  # for reference https://github.com/libimobiledevice/libimobiledevice/pull/335
+  patch :DATA
```

Comment:
Now you've submitted upstream does it work if we use https://github.com/libimobiledevice/libimobiledevice/pull/335.patch in a `patch do` format?